styles/pages/statistics.module.scss

```diff
@@ -0,0 +1,18 @@
+.main {
+	flex-grow: 1;
+	display: flex;
+	align-content: flex-start;
+	flex-wrap: wrap;
+	padding-block: 3.5vw;
+	column-gap: 4vw;
+	row-gap: 3.5vw;
+}
+
+@media only screen and (max-width: 800px) {
+	.main {
+		padding: 2rem;
+		justify-content: space-between;
+		column-gap: 2rem;
```

Comment:
if the column and row gap are the same, use the shorthand instead - `gap: 2rem;`

Reply:
noted